bin/cron/activity-monitor

```diff
@@ -205,7 +205,7 @@ def main
         level_url = "http://studio.code.org/s/#{sl.script.id}/level/#{sl.id}"
       end
       log_string += %[ <a href="#{browserstack_url(i[:browser], level_url) || level_url}">✈ Repro It</a>]
-      HipChat.notify('ActivityMonitor', log_string, color: 'yellow', notify: 1)
+      HipChat.message 'ActivityMonitor', log_string, color: 'yellow', notify: 1
```

Comment:
Because I (someone else) hasn't gotten around to renaming our code to reflect our move from HipChat to Slack. At the moment, calling `HipChat.message` sends the message to Slack.  :(